README.md

```diff
@@ -1,40 +1,55 @@
 # Nylas Java SDK
+This is the GitHub repository for the Nylas Java SDK and this repo is primarily for anyone who wants to make contributions to the SDK or install it from source. If you are looking to use Java to access the Nylas Email, Calendar, or Contacts API you should refer to our official [Java SDK Quickstart Guide](https://docs.nylas.com/docs/quickstart-java).
 
-The Nylas Java SDK provides all of the functionality of the Nylas [REST API](https://docs.nylas.com/reference) in an easy-to-use Java API. With the SDK, you can programmatically access an email account (e.g. Gmail, Yahoo, etc.) and perform functionality such as getting messages and listing message threads.
+The Nylas Communications Platform provides REST APIs for [Email](https://docs.nylas.com/docs/quickstart-email), [Calendar](https://docs.nylas.com/docs/quickstart-calendar), and [Contacts](https://docs.nylas.com/docs/quickstart-contacts), and the Java SDK is the quickest way to build your integration using Java.
 
-## Requirements
-- Java 8 or above
-- [Nylas developer account and application](https://docs.nylas.com/docs/get-your-developer-api-keys) 
+Here are some resources to help you get started:
 
-## Adding the SDK to your project
+- [Nylas SDK Tutorials](https://docs.nylas.com/docs/tutorials)
+- [Get Started with the Nylas Communications Platform](https://docs.nylas.com/docs/getting-started)
+- [Sign up for your Nylas developer account.](https://nylas.com/register)
+- [Nylas API Reference](https://docs.nylas.com/reference)
 
-For projects using Gradle, add the following to your dependencies section of build.gradle:
+If you have a question about the Nylas Communications Platform, please reach out to support@nylas.com to get help.
+
+# Install
+**Note:** The Nylas Java SDK requires Java 8 or above.
+
+**Install via Gradle**: If you're using Gradle, add the following to your dependencies section of build.gradle:
 
     implementation("com.nylas.sdk:nylas-java-sdk:1.X.X")
 
-For projects using Maven, add the following to your POM file:
+**Install via Maven**: For projects using Maven, add the following to your POM file:
 
     <dependency>
       <groupId>com.nylas.sdk</groupId>
       <artifactId>nylas-java-sdk</artifactId>
       <version>1.X.X</version>
     </dependency>
```

Comment:
Using the term "Install" feels a bit odd to me for using the SDK as a dependency, but I do see some people use and if that's the preferred term at Nylas, ok.

Reply:
I want to keep the top level header as Install, but I changed the subsections to Setup instead.